tasks/task03/ Word_stats.py

```diff
@@ -0,0 +1,16 @@
+import re
+texts = ["Hello, World!", "The world is mine", "Hello, how are you?"]
+dictw = {}
+i = 0
+for istr in texts:
+    for iword in re.split(r'[:; ,*.!?\n]+',istr.lower()):
+        if iword == "":
+            continue
+        if dictw.get(iword, "e404") == "e404":
+            dictw[iword] = [1, i]
+        else:
+            dictw[iword][0] += 1
+    i += 1
+print("word\t\tcount\tfirst line\n----------------------------------")
+for iword in dictw.keys():
```

Comment:
Nice trick here:
There is dict.iteritems() which returns key-value pairs:
for word, stats in dictw.iteritems():


Moreover, your value is a list, it supports unpacking, so you can write:
for word, (count, line) in dictw.iteritems()